kafka-ui-api/src/main/java/com/provectus/kafka/ui/model/InternalClusterState.java

```diff
@@ -40,7 +38,6 @@ public InternalClusterState(KafkaCluster cluster, MetricsCache.Metrics metrics)
         .orElse(null);
     topicCount = metrics.getTopicDescriptions().size();
     brokerCount = metrics.getClusterDescription().getNodes().size();
-    zooKeeperStatus = ClusterUtil.convertToIntServerStatus(metrics.getZkStatus().getStatus());
```

Comment:
ClusterUtil.convertToIntServerStatus method can be removed.

Reply:
done